docs/get_started.rst

```diff
@@ -1,17 +1,388 @@
-Get started
+Get Started
 ===========
 
+.. contents::
+    :depth: 2
+    :local:
+
+
+
 Install CumulusCI
 -----------------
 
+.. note:: These installation instructions assume some familiarity with entering commands into a terminal.
+    If that's completely new to you, we recommend visiting the `CumulusCI Setup module <https://trailhead.salesforce.com/content/learn/modules/cumulusci-setup>`_ on Trailhead for a step by step walkthrough.
+
+
+
+On macOS
+^^^^^^^^
+
+`Homebrew <https://brew.sh/>`_ is a prerequisite for installing CumulusCI on macOS. Start there.
+
+
+Install via ``pipx``
+********************
+
+``pipx`` ensures that CumulusCI and its dependencies are installed into their own Python environment separate from other Python software on your computer. **We cannot recommend it enough!**
+
+First, install ``pipx`` with the following commands::
+
+    $ brew install pipx
+    $ pipx ensurepath
+
+After ``pipx`` installs, install CumulusCI::
+
+    $ pipx install cumulusci
+
+Once finished `verify your installation`_.
```

Comment:
```suggestion
When finished, `verify your installation`_.
```
Style is to avoid using "once" to mean "after" or "when": https://confluence.internal.salesforce.com/display/DOCTEAM/once%2C+after%2C+when